src/main/java/me/contaria/seedqueue/mixin/compat/sodium/ChunkBuilderMixin.java

```diff
@@ -101,11 +112,12 @@ private ChunkBuildBuffers loadCachedBuildBuffersOnWall(@Coerce Object passId, Bl
     @Dynamic
     @Group(name = "loadCachedBuildBuffersOnWall")
     @WrapOperation(
-            method = "createWorker",
+            method = {"startWorkers", "createWorker"},
             at = @At(
                     value = "NEW",
                     target = "(Lme/jellysquid/mods/sodium/client/gl/attribute/GlVertexFormat;Lme/jellysquid/mods/sodium/client/render/chunk/passes/BlockRenderPassManager;)Lme/jellysquid/mods/sodium/client/render/chunk/compile/ChunkBuildBuffers;"
-            )
+            ),
+            require = 0, allow = 1
```

Comment:
Should this be "require = 1"? I see there was an older require = 0 on another method but I assume that this mixin would have failed before if the method was not present.

Reply:
I'm pretty sure this injection is for sodium mac only, so out of

old sodium, new sodium (this pr),
old sodium mac, new sodium mac (this pr ported)

it would still only succeed for 3 and 4.